Dockerfile

```diff
@@ -20,3 +22,7 @@ RUN gem install rake --version =12.3.0 \
     && gem install kitchen-terraform --version 3.0.0 \
     && gem install test-kitchen --version 1.16.0
 WORKDIR /tf-test/module
+
+RUN wget https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-amd64.tar.gz >/dev/null 2>&1
+RUN tar -zxvf go${GOLANG_VERSION}.linux-amd64.tar.gz -C /usr/local/ >/dev/null
+ENV PATH /usr/local/go/bin:/usr/local/bin:/usr/bin:$PATH
```

Comment:
Please don't forget to set the GOPATH, this is really important for Go environment.

Reply:
GOPATH varies between different modules because they have different folder names. Here I am going to set $HOME as GOPATH.